src/index.js

```diff
@@ -252,20 +252,21 @@ fflags.enabled('onetrust', () => {
     pinterest: /epik/,
     tiktok: /ttclid/,
   };
-  const params = Array.from(new URLSearchParams(window.location.search).keys());
-  Object.entries(networks).forEach(([network, regex]) => {
-    params.filter((param) => regex.test(param)).forEach((param) => sampleRUM('paid', { source: network, target: param }));
-  });
-})();
 
-fflags.enabled('email', () => {
-  const networks = {
+  const emailNetworks = {
     mailchimp: /mc_(c|e)id/,
     marketo: /mkt_tok/,
 
   };
-  const params = Array.from(new URLSearchParams(window.location.search).keys());
-  Object.entries(networks).forEach(([network, regex]) => {
-    params.filter((param) => regex.test(param)).forEach((param) => sampleRUM('email', { source: network, target: param }));
-  });
-});
+
+  const knownAdNetworksSection = matchParams(adNetworks, searchParams).map((network) => `paid=${network}`);
+  const knownEmailNetworksSection = matchParams(emailNetworks, searchParams).map((network) => `email=${network}`);
+  const utmSection = searchParams.filter(([key]) => key.startsWith('utm_'))
+    .filter(([key]) => key !== 'utm_id')
+    .filter(([key]) => key !== 'utm_term')
+    .map(([key, value]) => `${key}=${value}`);
+
+  const target = [knownAdNetworksSection, knownEmailNetworksSection, utmSection].join('&');
+
+  sampleRUM('acquisition', { source: document.referrer, target });
```

Comment:
we should only track if there is something of value
```suggestion
  target && sampleRUM('acquisition', { source: document.referrer, target });
```

Reply:
we can have an interesting `document.referrer` without any interesting query params. One example is `https://www.youtube.com` 

when the document.referrer is youtube, then it is always `paid:video:youtube` regardless of the query params: https://github.com/adobe/helix-rum-collector/pull/347#pullrequestreview-2150947738

Reply:
You don't get to see the referrer's URL parameters. For Youtube, I'd start with the assumption of owned, unless there are tracking parameters that indicate otherwise.